bin/node/Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ['Joystream contributors']
 build = 'build.rs'
 edition = '2018'
 name = 'joystream-node'
-version = '8.0.0'
+version = '7.4.1'
```

Comment:
Strangely looks like the changes from https://github.com/Joystream/joystream/pull/4428 were reverted